ignite/metrics/clustering/silhouette_score.py

```diff
@@ -0,0 +1,121 @@
+from typing import Any, Callable, Union
+
+import torch
+from torch import Tensor
+
+from ignite.metrics.clustering._base import _ClusteringMetricBase
+
+__all__ = ["SilhouetteScore"]
+
+
+def _get_silhouette_score(**kwargs: Any) -> Callable[[Tensor, Tensor], float]:
```

Comment:
Can you please change this function as well

Reply:
Thanks for review.

I left this function in the previous style for retaining extra kwargs for sklearn's silhouette score function.

Should we make kwargs as a member of the metric class as in [Cohen Kappa's weight argument](https://github.com/pytorch/ignite/blob/a5d3464231f37b86fe4e5ae272fe14bbcb33b27d/ignite/metrics/cohen_kappa.py#L71-L87)?

Reply:
I see, thanks for pointing out this. Let's use one approach everywhere, I let you choose either this one or CohenKappa's one

Reply:
I'll take the CohenKappa's style for simplicity.